Trace/src/V2/resources/trace_service_rest_client_config.php

```diff
@@ -17,7 +17,7 @@
             ],
             'CreateSpan' => [
                 'method' => 'post',
-                'uriTemplate' => '/v2/{name=projects/*/traces/*}/spans',
+                'uriTemplate' => '/v2/{name=projects/*/traces/*/spans/*}',
```

Comment:
fixing something broken or breaking something?

Reply:
Good call out! It looks like this is a fix.